woocommerce/woocommerce-android
# [Tablet Support M3] Part 8 – Disable Create button if recalculate is needed

Closes: #11103 


### Description

This PR adds a few improvements to the order creation flow UI in tablet layout:
1. Disables the "Create" button in the Order Summary pane in case recalculate is required (Recalculate button is visible).
2. Enforces recalculation when switching to a single pane from a dual-pane layout – to avoid loosing item selection state.

### Testing instructions

1. Select items in dual pane layout (e.g. on tablet) and observe Create button is disabled until recalculation is triggered.
2. Select items in dual pane layout, and without clicking the Recalculate button, reduce screen size (e.g. open another app in split mode) so that only the single pane with Order Summary is visible. Observe recalculation is triggered – the items are added to the order, the order is synced and the Collect payment button is visible.
